notifications/metrics.go

```diff
@@ -5,6 +5,18 @@ import (
 	"fmt"
 	"net/http"
 	"sync"
+
+	prometheus "github.com/docker/distribution/metrics"
+	"github.com/docker/go-metrics"
+)
+
+var (
+	// eventsCounter counts total events of incoming, success, failure, and errors
+	eventsCounter = prometheus.NotificationsNamespace.NewLabeledCounter("events", "The number of total events", "type", "to")
```

Comment:
Non-blocking nit: How about changing label name from `to` to `endpoint` instead? That terminology is consistent with codebase and config. Sorry I should've realized this before suggesting the name long time ago.